Mathlib/RingTheory/Coalgebra.lean

```diff
@@ -0,0 +1,97 @@
+/-
+Copyright (c) 2023 Ali Ramsey. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Ali Ramsey
+-/
+import Mathlib.RingTheory.TensorProduct
+import Mathlib.LinearAlgebra.TensorProduct
+
+/-!
+# Definition and example of a coalgebra
+-/
+
```

Comment:
```suggestion

suppress_compilation

```
will fix the error below.

Reply:
Ah ok thanks. Why was I getting that error, out of interest?

Reply:
We disabled compilation (i.e., making actual real programs that run) for tensor products because it took too long, and we were never using the output anyway. The error is telling you "this function was never compiled".